### PR TITLE
meta-mender-tegra: kas related fixes and updates 

### PR DESCRIPTION
**Pull request by TheYoctoJester**
This merges
- the Readme update to not use the repo based setup mechanism anymore, and moves to kas
- the fix of an unintentional layer dependency duplication, resulting in a non-reproducible floating revision.

### CODE REVIEW

**Comment by TheYoctoJester** [comment]
@dwalkes to prepare `kirkstone` for some of the things you commented on. Thoughts?

**Comment by dwalkes** [comment]
@TheYoctoJester looks good, I haven't given the kas config a test drive, will try to kick that off tonight, but probably no need to wait for that.

File: kas/include/tegra.yml [hunk 1]
```diff
@@ -10,8 +10,6 @@ repos:
     refspec: kirkstone-l4t-r32.7.x
 
   meta-openembedded:
-    url: https://git.openembedded.org/meta-openembedded
-    refspec: kirkstone
```
Comment (kacf) [inline kas/include/tegra.yml:14]: Nitpick: The commit only says that this is removed, but not why (and it's not clear to me).

**Review by kacf — APPROVED** [review]

File: kas/include/tegra.yml (hunk 1 above)
Comment (TheYoctoJester) [reply kas/include/tegra.yml:14]: Very good catch. Addressed in update, okay with it?

Comment (kacf) [reply kas/include/tegra.yml:14]: 👍

**Pull request merged**